### ISSUE TITLE
False Positive Relative Import

### ISSUE THREAD

**Issue by hiranya911** [PyCQA/pylint#1489]
### Steps to reproduce
1. Write a source file with an absolute import like `import google.auth` (where `google.auth` refers to a third-party lib)
2. Run pylint on the source file

### Current behavior
`W: 19, 0: Relative import 'google.auth', should be 'my_package.google.auth' (relative-import)`

### Expected behavior
This should not be a warning. 

### pylint --version output
pylint 1.7.1, 
astroid 1.5.2
Python 2.7.6 (default, Mar 17 2017, 18:16:39) 
[GCC 4.8.4]

This worked fine in pylint 1.6.4

**Comment by katyukha** [comment]
Seems that this is not fixed yet.

This bug is still reproducable if pylint runs from virtualenv.

For example we have code like mentioned in issued description.
If we ran system pylint, everything is OK
but if we install pylint in virtualenv, and run it, it is still have false positive on relative imports.

Tested on pylint version:
```
pylint 1.7.2,   also tested [1.7.1, 1.7.0]
astroid 1.5.3
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
```

```
virtualenv --version
15.1.0
```
But pylint version 1.6.5 works fine.

**Comment by PCManticore** [comment]
@katyukha We didn't release this yet.

**Comment by hiranya911** [comment]
Is there a timeline for the release of this fix? I can still repro on pylint 1.7.4.

**Comment by PCManticore** [comment]
Yes, this is going to be part of the 1.8 release, which is the code from the master. There is no timeline for the release though, but it will definitely be in the following last months of the year.
